masschallenge/calendar-module
# AC-7073 :: Calendar picker for individual office hours holder availability

### Changes introduced in: [AC-7073](https://masschallenge.atlassian.net/browse/AC-7073):
- Add a calendar picker for individual office hours holder availability
### How to test
- Checkout this branch on ` impact and frontend`
- On localhost
- In `front-end` run `yarn`
- Navigate to this mentor's office hours [page](http://localhost:8181/officehours/bos/2019/?mentor_id=59788) and schedule office hours on his behalf.
- Masquerade as this [finalist](http://localhost:8181/admin/simpleuser/user/13884/masquerade/)
- Visit http://localhost:1234/directory/people/56778
- Click the `Available Office Hours` button
- See that a modal show up with the calendar.
- Follow the mockups attached on this ticket and ensure that the feature works as required.
